docs/lib/DEBUGGING.md

```diff
@@ -144,6 +144,20 @@ The OpenSDA port provides a debugger interface, which is compatible with the Seg
 
 To use the JTAG port instead, download the tools and drivers for the debugger of your choice.
 
+#### ULINK pro configuration for NXP FRDM-K64F
+
+Configure NXP FRDM-K64F according to the pdf document from arm Keil [Application Note](http://www.keil.com/appnotes/docs/apnt_287.asp).
+
+In order to add an example to uVision run the "mbed export -i UVISION5 -m K64F" command, and open the project from uVision with the .uvprojx file created.
+For more information check [Exporting to uVision](https://developer.mbed.org/handbook/Exporting-to-uVision).
+
+Most ULINK pro features configuration are described in the Application Note except Instruction trace that is [not officially supported](http://www.keil.com/support/man/docs/ulinkpro/ulinkpro_trace_instruction_trace.htm) with cortex-M devices, we can bypass that with the PC SAMPLING option (as described in the image below)
```

Comment:
Please change to "The application note describes the configuration for most ULINKpro features. It does not include a description of Instruction trace, which cortex-M devices [do not officially support](http://www.keil.com/support/man/docs/ulinkpro/ulinkpro_trace_instruction_trace.htm). You can bypass this with the `PC SAMPLING` option, which the image below describes."